### PR TITLE
Proxy review comments

### PR DESCRIPTION
**Pull request by dlespiau**
I've done a few tweaks to address James review comments that weren't captured as their own issue.

In doing so, discovered a race in the mock hyperstart that I fixed.

### CODE REVIEW

File: proxy/README.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 `cc-proxy` is a daemon offering access to the
 [`hyperstart`](https://github.com/hyperhq/hyperstart) VM agent to multiple
-clients on the host.
+clients. Only a single instance of cc-proxy per host is necessary as it can be
```
Comment (jodh-intel) [inline proxy/README.md]: `cc-proxy` rather than "cc-proxy".

**Comment by dlespiau** [comment]
PR updated:

  - Rebased onto master
  - Added missing backticks for a cc-proxy string in README.md (James)

**Comment by jodh-intel** [comment]
Hi @dlespiau - this looks good, but the branch needs to be rebased.

**Comment by dlespiau** [comment]
Updated PR:

  - Rebased branch on top of Sébastien hyperstart mock changes.

**Comment by grahamwhaley** [comment]
lgtm Will await further acks before merging etc.

**Comment by jodh-intel** [comment]
lgtm.

**Comment by chavafg** [comment]
qa-failed

**Comment by dlespiau** [comment]
Updated PR:

  - Rebased and solved conflicts with the go linter branch that was just merged

**Comment by dlespiau** [comment]
This needs PR https://github.com/01org/cc-oci-runtime/pull/378 + rebase to pass the tests again

**Comment by dlespiau** [comment]
Updated PR: rebased once again

**Comment by grahamwhaley** [comment]
given this is just a rebase, re-adding my
lgtm

**Comment by gorozco1** [comment]
lgtm

**Comment by chavafg** [comment]
qa-failed

**Comment by chavafg** [comment]
again false negative on functional tests, moving to qa-passed

**Comment by chavafg** [comment]
qa-passed

**Pull request merged**